cli/src/commands/operation/undo.rs

```diff
@@ -70,5 +71,26 @@ pub fn cmd_op_undo(
     tx.repo_mut().set_view(new_view);
     tx.finish(ui, format!("undo operation {}", bad_op.id().hex()))?;
 
+    if let Some(mut formatter) = ui.status_formatter() {
+        write!(formatter, "Undid operation ")?;
+        let workspace_env = workspace_command.env();
+        let language = OperationTemplateLanguage::new(
+            bad_op.op_store().root_operation_id(),
+            Some(bad_op.id()),
```

Comment:
this should be `workspace_command.repo().op_id()`. It's usually the operation specified by `--at-op`, and here we've already committed the transaction, the operation after undo is the current operation.

Reply:
`workspace_command.repo().op_id()` is not a thing (`op_id()` on the repo specifically). `bad_op` should be correct though here no? It is the result parsed from the input `    let bad_op = workspace_command.resolve_single_op(&args.operation)?;`

Reply:
No, it's not. The `current_op_id` argument will be used in order to test if the context operation (= `bad_op`) is the head of the operation at the currently loaded operation (= `repo.operation()`).

`bad_op` is the operation to be reverted.

Reply:
Oh right